File: loops.py
""" Loops o Ciclos """

""" 

for value in  collection:
    sentences

"""

for i in range(1, 100, 2): # start = 0, stop = valor a detener, step = 1
    print(i) # 1 3 5 7 9
    
nombres = ["Hugo", "Paco", "Luis"]

for i in range(len(nombres)):
    print(nombres[i])
    
for nombre in nombres:
    print(nombre) # Hugo Paco Luis
    
    
indice = 0
while(indice < len(nombres)):
    print(nombres[indice])
    #indice = indice + 1
    indice += 1

File: loops.js
 /* Loops o Ciclos */
 /* 
 
 for(counter; condition; increment){
    sentences
 }

 for (indice in collection){
    sentences
 }

 for (value in collection){
    sentences
 }

 while (condition){
    sentences
 }

 do {
    sentences
 } while (condition)
 
 
 */

 for(let i = 1;  i <= 10; i++){
    console.log(i)
 }

 let nombres = ["Hugo", "Paco", "Luis"]
 for(let indice = 0; indice < nombres.length; indice++){
    console.log(nombres[indice])
 }

 for(let indice in nombres){
    console.log(indice) // 0 1 2 
    console.log(nombres[indice])
 }

 for(let nombre of nombres){
    console.log(nombre) // Hugo Paco Luis
 }

 let i = 1;
 while(i <= 10){
    console.log(i)
    i++
 }

 let indice = 0;
 while(indice < nombres.length){
    console.log(nombres[indice])
    indice++
 }

 i = 1;
 do {
    console.log(i);
    i++
 } while (i <= 10)